docs/use_cases/interoperability.rst

```diff
@@ -53,3 +53,14 @@ such an explicit conversion::
 
         using namespace std::chrono_literals;
         quantity q = 1s;  // ERROR
+
+For external quantity point-like types, `quantity_point_like_traits` is also provided.
```

Comment:
Please add `quantity_point_like_traits` to `customization_points.rst` or otherwise Sphinx will complain about unresolved link.